packages/gatsby-theme-carbon/gatsby-config.js

```diff
@@ -14,6 +14,7 @@ module.exports = themeOptions => {
     repository = {
       baseUrl: '',
       subDirectory: '',
+      branch: 'master',
```

Comment:
@vpicone wouldn't this be setting the default `themeOptions` to master?

Reply:
I'm confused by the docs update

>Even if you have no `subDirectory` an empty string `''` must be supplied.

Can you elaborate on that? Unfortunately we can't force folks to add a theme option without it being a breaking change/major release.

Reply:
So to clarify this is a pre-existing issue.  They are not required to put in that theme option unless they want the edit on Github link.  In which case you would be required to supply all nodes of the `repository` option.

This was something I found because my site does not have a subdirectory (is a user site) and if I do not have that `subDirectory: ''` the site does not build with the following error.

```
 ERROR #85907  GRAPHQL

There was an error in your GraphQL query:

- Unknown field 'subDirectory' on type 'SiteSiteMetadataRepository'.

File: node_modules/gatsby-theme-carbon/src/components/EditLink/EditLink.js

⠙ extract queries from components
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

Maybe my documentation addition needs to be reworded, and I can open another bug issue.